lldb/test/API/tools/lldb-dap/stackTrace-x86/TestDAP_source_x86.py

```diff
@@ -0,0 +1,64 @@
+"""
+Test lldb-dap stack trace containing x86 assembly
+"""
+
+import lldbdap_testcase
+from lldbsuite.test.decorators import skipUnlessArch, skipUnlessPlatform
+from lldbsuite.test.lldbtest import line_number
+
+
+class TestDAP_stacktrace_x86(lldbdap_testcase.DAPTestCaseBase):
+    @skipUnlessArch("x86_64")
+    @skipUnlessPlatform(["linux"])
```

Comment:
is this really limited to linux? It seems like this should work on at least  Darwin too.

Reply:
seems like it should work, changing